src/computed/channel.js

```diff
@@ -71,6 +71,13 @@ const ComputedChannel = store => {
         .map(c => Number(c.localBalance))
         .reduce((a, b) => a + b, 0);
     },
+    get channelBalanceOnChainLabel() {
+      const { confirmedBalanceSatoshis, settings } = store;
```

Comment:
@tanx We want `confirmedBalanceSatoshis + unconfirmedBalanceSatoshis`, right? Just everything that's not in channels?

Reply:
Yep. You can use `store.balanceSatoshis` here for the total on-chain balance.